src/amo/components/Page/index.js

```diff
@@ -39,13 +43,12 @@ export const PageBase = ({
         <div
           className={makeClassName('Page', {
             'Page-not-homepage': !isHomePage,
+            'Page-no-hero-promo': !enableFeatureHeroRecommendation,
```

Comment:
I don't think this is a great way to control the style changes because the state of the config flag doesn't necessarily mean you need to add the styles. I offered a more direct way to control the style changes.

Reply:
Your suggestion addresses the hacky style I added for the error message padding, but it doesn't address the need for the `max-width` at https://github.com/mozilla/addons-frontend/pull/8753/files#diff-5bb54930f8afd38fcb0856c0b4dedc86R25, so I've kept this in place for now. I can do a follow-up later this week if needed.